src/map/navmesh.cpp

```diff
@@ -96,6 +96,16 @@ void CNavMesh::ToDetourPos(const position_t* pos, float* out)
     out[2] = z * -1;
 }
 
+CNavMesh::CNavMesh(CNavMesh* other)
+: m_zoneID(other->m_zoneID)
+, m_navMesh(other->m_navMesh)
+{
+    std::memset(&m_hitPath, 0, sizeof(m_hitPath));
+
+    m_hit.path    = m_hitPath;
+    m_hit.maxPath = 20;
+}
+
 CNavMesh::CNavMesh(uint16 zoneID)
```

Comment:
I would ask Shozo if this is gonna effect anything in the new navmesh code

Reply:
Yes, this will not merge directly.